src/libpgmoneta/wf_restore.c

```diff
@@ -380,13 +311,15 @@ combine_incremental_execute(char* name, struct art* nodes)
    pgmoneta_log_debug("Combine incremental (execute): %s/%s", config->common.servers[server].name, identifier);
    bck = (struct backup*)pgmoneta_art_search(nodes, NODE_BACKUP);
 
-   prior_backups = (struct deque*)pgmoneta_art_search(nodes, NODE_BACKUPS);
-   if (prior_backups == NULL || pgmoneta_deque_size(prior_backups) < 2)
+   prior_labels = (struct deque*)pgmoneta_art_search(nodes, NODE_LABELS);
+   label = (char*) pgmoneta_art_search(nodes, NODE_LABEL);
+   if (prior_labels == NULL || pgmoneta_deque_size(prior_labels) < 1)
    {
-      pgmoneta_log_error("Combine incremental: should have at least 2 backups");
+      pgmoneta_log_error("Combine incremental: should have at least 1 backup");
```

Comment:
Eh, what ?

Reply:
It used to include the current backup, now it's just previous ones, so at least one previous backup in the chain

Reply:
Updated message to make it clearer.